### DISCUSSION TITLE
Compatibility issue with custom AxiosRequestConfig and axios version 1.2.3

### DISCUSSION THREAD

**Discussion by Sajarin-M** [axios/axios#5477] [Q&A]
I have extended the axios module in my project by adding a custom property, `showNotification`, to the `AxiosRequestConfig` interface as shown below:
```ts
declare module 'axios' {
  export interface AxiosRequestConfig {
    showNotification?: boolean;
  }
}
```
It was running without any issues before. However, after updating to version `1.2.3` of `axios`, I am now encountering a TypeScript error stating that:
```ts
Argument of type '{ showNotification: boolean; }' is not assignable to parameter of type 'RawAxiosRequestConfig<any>'
```
I did not intentionally update my packages and it seems that it has been installed automatically as a patch release.

I am wondering if this is a breaking change that occurred in the patch release? If so, what is the proper way to include my custom `AxiosRequestConfig` type declaration while still being compatible with the new version of `axios`?

**Comment by jasonsaayman** [comment]
Please try the latest version should this still occur open a new issue
